pkg/codec/chain_modifier.go

```diff
@@ -0,0 +1,38 @@
+package codec
+
+import (
+	"reflect"
+)
+
+type ChainModifier []Modifier
```

Comment:
does modifierChainer name make more sense in our codebase? Naming anything with chain adds a subconscious expectation 😄

Reply:
still has chain in chainer.  How about multiModifier?

Reply:
sure: 👍🏼